packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -114,13 +96,29 @@ export class Tooltip<
         this.popoverRef.current?.reposition();
     }
 
+    protected validateProps(props: TooltipProps<T>) {
+        const childrenCount = React.Children.count(props.children);
+        if (childrenCount > 1) {
+            console.warn(Errors.POPOVER_WARN_TOO_MANY_CHILDREN);
+        }
+        // all other warnings should occur in Popover, not here.
+    }
+
     // any descendant ContextMenus may update this ctxState
     private renderPopover = (ctxState: TooltipContextState) => {
-        const { children, compact, disabled, intent, popoverClassName, ...restProps } = this.props;
+        const { children, content, renderTarget, compact, disabled, intent, popoverClassName, ...restProps } =
+            this.props;
+
         const popoverClasses = classNames(Classes.TOOLTIP, Classes.intentClass(intent), popoverClassName, {
             [Classes.COMPACT]: compact,
         });
 
+        const tooltipId = Utils.uniqueId("tooltip");
+
+        const childTarget = Utils.ensureElement(React.Children.toArray(children)[0], undefined, {
+            "aria-describedby": tooltipId,
```

Comment:
could we try some version of https://github.com/facebook/react/issues/8205#issuecomment-755866890 and checking [hasOwnProperty](https://github.com/facebook/react/issues/8205#issuecomment-334172163) to make sure we're not overriding an id specified by a consumer of Tooltip?

Reply:
Actually `aria-describedby` can have multiple values as a space separated list string (https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby). Updated to concatenate that if passed: https://github.com/palantir/blueprint/pull/6865/commits/730d177bfaa936d6ef973f7fb0e83757e7b889ce